universal-login-react/src/ui/commons/GasPrice/GasPrice.tsx

```diff
@@ -62,10 +62,9 @@ export const GasPrice = ({isDeployed = true, deployedWallet, sdk, gasLimit, onGa
   };
 
   useEffect(() => {
-    const fastGasModeIndex = 1;
     if (gasModes) {
-      const {name, usdAmount} = gasModes[fastGasModeIndex];
-      const gasOption = findGasOption(gasModes[fastGasModeIndex].gasOptions, ETHER_NATIVE_TOKEN.address);
+      const {name, usdAmount} = gasModes[FAST_GAS_MODE_INDEX];
```

Comment:
I think is better than introduce FAST_GAS_MODE_INDEX. It add for us flexibility. (we also can change gasModesName to enums)
const gasMode= findGasMode(gasModes, 'fast')

Reply:
Why not just `defaultGasMode(gasModes)`?